### PR TITLE
Encode log.origin.file.* correctly

### LINKED ISSUE #7: `log.origin.file.name` format is invalid
When `ReportCaller` is enabled, the formatter will map caller information to `log.origin.*`. Logrus combines the filename and line number together, and we are recording this combined value in `log.origin.file.name`. e.g. `"log.origin.file.name": "/home/andrew/go/src/github.com/elastic/ecs-logging-go-logrus/example_test.go:48"`

We should be recording the filename and line number separately, in `log.origin.file.name` and `log.origin.file.line` respectively.

### PR DESCRIPTION
**Pull request by axw**
Encode caller information explicitly, rather than
relying on Logrus to do it. Logrus combines file:line
into one field, whereas we want to encode them as two
separate fields.

Closes #7

### CODE REVIEW

File: formatter.go [hunk 1]
```diff
@@ -91,9 +91,43 @@ func (f *Formatter) Format(e *logrus.Entry) ([]byte, error) {
...
+			data["log.origin.file.name"] = fileVal
+		}
+		if lineVal > 0 {
+			data["log.origin.file.line"] = lineVal
+		}
+	}
 	data["ecs.version"] = ecsVersion
 	ecopy := *e
 	ecopy.Data = data
+	ecopy.Caller = nil
```
Comment (simitt) [inline formatter.go:130]: Why is this necessary when you set `e.Caller = nil` in L116?

**Review by simitt — APPROVED** [review]

File: formatter.go (hunk 1 above)
Comment (axw) [reply formatter.go:130]: It's not, thanks!
